manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+PackageIdentifier: HiroSystems.Clarinet
+PackageVersion: 0.14.1
+PackageLocale: en-US
+Publisher: Hiro Systems
+PublisherUrl: https://www.hiro.so/
+PrivacyUrl: https://www.hiro.so/p/terms-privacy
+PackageName: Clarinet
```

```suggestion
PackageName: clarinet
```
must match Control Panel.